### PR TITLE
changed the file name to .gitignore. In index.html added metategs, li…

### PR DESCRIPTION
**Pull request by Irina-pi**
…sts, tables, forms.

I probably got tangled in the branches. I made a mistake and tried to fix it. But this is not accurate.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,13 +2,138 @@
...
   </head>
   <body>
     <h1>Моя первая страница</h1>
-    <hr />
+    <hr>
     <p>
       <a href="Hello/about.html">об авторе</a>
     </p>
+    <img src="./images/flower.jpg" alt="flowers">
+    <b>
```
Comment (5tronciy) [inline index.html]: `<ol>` is not phrasing content. Don't use `<b>` to wrap the blocks.
Read more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b)

File: index.html [hunk 2]
```diff
@@ -2,13 +2,138 @@
...
+      <tr>
+        <td>&#60;tfood&#62;</td>
+        <td>Groups the footer content in a table</td>
+      </tr>
+      </tbody>
+    </table>
+    <form>
+      <fieldset>
+        <legend>My first form</legend>
+        <p>
```
Comment (5tronciy) [inline index.html]: The same as `<b>`. Use `<div>` to wrap the blocks.

File: index.html [hunk 3]
```diff
@@ -2,13 +2,138 @@
...
+          <label for="name">Name</label><br />
+          <input id="name" type="text" name="name" placeholder="Name" /><br>
+          <label for="email">Email</label><br />
+          <input id="email" type="email" name="email"/><br>
+          <label for="password">Password</label><br />
+          <input id="password" type="password" name="Password"/><br>
+          <label for="programminglevel">Your programming level</label><br />
+          <input id="programminglevel" type="range" name="programminglevel"/>
```
Comment (5tronciy) [inline index.html]: The default range doesn't match the task. Set it up yourself.

File: index.html [hunk 4]
```diff
@@ -2,13 +2,138 @@
...
+          What does HTTP stand for?
+        </p>
+        <input id="answer-1" type="radio" name="question" value="1">
+        <label for="answer-1">Hyper Text Translate Protokol</label><br />
+        <input id="answer-2" type="radio" name="question" value="2">
+        <label for="answer-2">Hyper Text Transmission Protocol</label><br />
+        <input id="answer-3" type="radio" name="question" value="3">
+        <label for="answer-3">Hyper Text Transfer Protocol</label><br />
+        <label for="comment">Comment</label><br>
+        <textarea id="comment" name="comment" placeholder="Comment"></textarea><br>
```
Comment (5tronciy) [inline index.html]: The length of text doesn't match the task.

File: index.html [hunk 5]
```diff
@@ -2,13 +2,138 @@
...
+        <label for="select answer">What does HTML stand for?</label><br />
+        <select name="answer" id="select answer">
```
Comment (5tronciy) [inline index.html:125]: Notes about id:
The id name is case sensitive!
The id name must contain at least one character, cannot start with a number, and must not contain whitespaces (spaces, tabs, etc.).

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Looks good. But need to be worked at.

File: index.html (hunk 1 above)
Comment (Irina-pi) [reply index.html]: I corrected.

File: index.html (hunk 2 above)
Comment (Irina-pi) [reply index.html:95]: Do I need to replace <p>?

Comment (Irina-pi) [reply index.html]: I corrected.

File: index.html (hunk 4 above)
Comment (Irina-pi) [reply index.html]: I corrected.

Comment (Irina-pi) [reply index.html]: I have added the cols="22" attribute to the textarea. did I do right?

File: index.html (hunk 5 above)
Comment (Irina-pi) [reply index.html:125]: I corrected.

File: index.html (hunk 3 above)
Comment (Irina-pi) [reply index.html]: I corrected.

File: index.html (hunk 4 above)
Comment (5tronciy) [reply index.html]: Hmm, sure it is not limit the textarea content to 500 characters.

File: index.html (hunk 5 above)
Comment (5tronciy) [reply index.html:125]: You didn't. Space is not allowed in id.

File: index.html [hunk 6]
```diff
@@ -7,6 +7,7 @@
     <meta name="autor" content="Anton and Irina Bocharov">
     <meta name="vienport" content="width=device-width, initial-scale=1.0">
     <title>Моя первая страница</title>
+    <link rel="stylesheet" href="./styles/styles.css">
```
Comment (5tronciy) [inline index.html:10]: Create new branch for the next task pls.

**Review by 5tronciy — CHANGES_REQUESTED** [review]

File: index.html (hunk 5 above)
Comment (Irina-pi) [reply index.html:125]: I understand! Thank you!

File: index.html (hunk 6 above)
Comment (Irina-pi) [reply index.html:10]: ok

**Pull request merged**